openid-connect-server/src/test/java/org/mitre/openid/connect/service/impl/TestMITREidDataService_1_1.java

```diff
@@ -1207,16 +1207,16 @@ public void testExportGrants() throws IOException {
                 } else {
                     assertThat(site.get("whitelistedSiteId").getAsLong(), equalTo(compare.getWhitelistedSite().getId()));
                 }
-                if (site.get("approvedAccessTokens").isJsonNull() || site.getAsJsonArray("approvedAccessTokens") == null) {
-                    assertTrue(compare.getApprovedAccessTokens() == null || compare.getApprovedAccessTokens().isEmpty());
-                } else {
-                    assertNotNull(compare.getApprovedAccessTokens());
-                    Set<String> tokenIds = new HashSet<String>();
-                    for(OAuth2AccessTokenEntity entity : compare.getApprovedAccessTokens()) {
-                        tokenIds.add(entity.getId().toString());
-                    }
-                    assertThat(jsonArrayToStringSet(site.getAsJsonArray("approvedAccessTokens")), equalTo(tokenIds));
-                }
+//                if (site.get("approvedAccessTokens").isJsonNull() || site.getAsJsonArray("approvedAccessTokens") == null) {
```

Comment:
I definitely appreciate you taking the time to look into the issue and submitting a patch for it! You can actually just push additional commits to the same branch and the pull request will be updated with the latest commits. Changing a fundamental data model is going to touch a lot of different places throughout the code and will by no means be simple to make sure it's fully functional.